assistant_dists/dream_kg_prompted/docker-compose.override.yml

```diff
@@ -0,0 +1,299 @@
+services:
+  agent:
+    command: sh -c 'bin/wait && python -m deeppavlov_agent.run agent.pipeline_config=assistant_dists/dream_kg_prompted/pipeline_conf.json'
+    environment:
+      WAIT_HOSTS: "spacy-nounphrases:8006, sentseg:8011, ranking-based-response-selector:8002, 
+          ner:8021, entity-linking:8075, combined-classification:8087, entity-detection:8103, terminusdb-server:6363, 
+          user-knowledge-memorizer:8027, property-extraction:8131, custom-entity-linking:8153, dff-knowledge-prompted-skill:8050, 
+          sentence-ranker:8128, prompt-selector:8135"
+      WAIT_HOSTS_TIMEOUT: ${WAIT_TIMEOUT:-1000}
+      HIGH_PRIORITY_INTENTS: 1
+      RESTRICTION_FOR_SENSITIVE_CASE: 1
+      ALWAYS_TURN_ON_ALL_SKILLS: 0
+      LANGUAGE: EN
+      FALLBACK_FILE: fallbacks_dream_en.json
+
+  spacy-nounphrases:
```

Comment:
i think you do not need this component

Reply:
cause spacy_nounphrases are used by NER & Entity Detection, and we need NER & Entity Detection inside Property Extraction as a source of `named_entities` and `entities_with_labels`, see here:

https://github.com/deeppavlov/dream/blob/50768c71ed6521f5f107c3df2328e52d5f4b6be5/annotators/property_extraction/server.py#L157C4-L158C100

`` named_entities_batch = request.json.get("named_entities", [[] for _ in init_uttrs])
    entities_with_labels_batch = request.json.get("entities_with_labels", [[] for _ in init_uttrs])``